wordpress-mobile/WordPress-iOS
# Fix exit animation of notices

Fixes #18019

## Description
- Fixed issue that caused the change in vertical position of notices to not be animate on dismissal 7604a49983e712ca4024d2caebeef2b392d50e49
- Modified the dismissal animation to move the whole notice off-screen instead of moving it to the bottom of the screen.

Normal Animation Speed

https://user-images.githubusercontent.com/25306722/159398891-c9d13c93-962d-4257-8506-be3727be60a4.mp4

Slowed down Animation Speed

https://user-images.githubusercontent.com/25306722/159398962-0747b6b8-8ce4-4384-a82b-19bdc2933b39.mp4


## Testing Instructions
1. Enable quick start for a site
2. Go back to My Site
3. Tap Customize Your Site
4. Tap on a step and follow-through
5. Observe the animation of replacing a notice with a new notice; it should be smooth. 
6. When done with a step, a notice for the next step will be automatically shown
7. Press Not now
8. Observe the notice being dismissed with a smooth animation

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
